src/Mvc/Mvc.Core/src/ProducesResponseTypeAttribute.cs

```diff
@@ -43,6 +55,19 @@ public ProducesResponseTypeAttribute(Type type, int statusCode)
         /// </summary>
         public int StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the default error type should be used when one
+        /// is expliclty provided as a type in the <see cref="ProducesResponseTypeAttribute"/>.
+        ///
+        /// When <see langword="true"/>, the ApiExplorer will use the provided <see cref="Type"/>
+        /// as the default type for error objects.
+        ///
+        /// When <see langword="false"/>, the ApiExplorer will use the globally configured default
+        /// error type.
+        /// </summary>
+        /// <value></value>
+        public bool OverrideDefaultErrorType { get; set; }
```

Comment:
Open to alternative ideas for names here. My creativity puttered and this was the most explicit name that I could come up with.

Reply:
I think I'll play around with this to get a sense for the bug. Declaring the error behavior here is too far disconnected from the implementation that configures one to make sense.

Reply:
Another approach is to change the below:

https://github.com/dotnet/aspnetcore/blob/b98bceaf22d55639fafebc52907c041d0cbacbc1/src/Mvc/Mvc.Core/src/ApplicationModels/ApiBehaviorApplicationModelProvider.cs#L45-L46

to read from an option other than `SuppressMapClientErrors`.